tst/CTA.Rules.Test/CTATests.cs

```diff
@@ -210,6 +208,18 @@ public void TestMonolithReplacementsWebAPI()
             StringAssert.Contains(@"return Content(MonolithService.CreateRequest(Request, this.ControllerContext.RouteData));", houseControllerText);
         }
 
+        [Test]
+        public void TestProjectOutsideSolutionFolder()
+        {
+            var results = AnalyzeSolution("Application_Proj_diff_folder.sln", tempDir, downloadLocation, version);
+
+            var csProjContent1 = results.ProjectResults[0].CsProjectContent;
+            var csProjContent2 = results.ProjectResults[1].CsProjectContent;
+
+            Assert.True(csProjContent1.IndexOf(string.Concat(">", version, "<")) > 0);
+            Assert.True(csProjContent2.IndexOf(string.Concat(">", version, "<")) > 0);
+        }
```

Comment:
nit: Can we rename the method to give more context for what the test is checking for? For example, `TestProjectsOutsideSolutionPath_Are_Ported` or similar.

Also, if the test is renamed, it may make sense to change the assertions to better describe the behavior we're looking for, for example: 
* Assert that the project files exist 
* Assert that the project file contents contain `version`

Again, these are nits but I think they will help improve readability- let me know if you think differently!

Reply:
Agreed yup.